app/src/main/java/com/danielkim/soundrecorder/fragments/RecordFragment.java

```diff
@@ -76,10 +74,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View recordView = inflater.inflate(R.layout.fragment_record, container, false);
 
-        mChronometer = (Chronometer) recordView.findViewById(R.id.chronometer);
         //update recording prompt text
         mRecordingPrompt = (TextView) recordView.findViewById(R.id.recording_status_text);
 
+        mCountButton=(Button)recordView.findViewById(R.id.btnCount);
```

Comment:
It's better to separate this change and create a new Pull Request. Please keep the existing PR clean and (maybe) it will be accepted by the repository owner (someday).